runtime/errors.go

```diff
@@ -211,6 +211,44 @@ func (e *ParsingCheckingError) Error() string {
 	return e.Err.Error()
 }
 
-func (e ParsingCheckingError) Unwrap() error {
+func (e *ParsingCheckingError) Unwrap() error {
 	return e.Err
 }
+
+func (e *ParsingCheckingError) ImportLocation() common.Location {
+	return e.Location
+}
+
+// InvalidContractDeploymentError
+//
+type InvalidContractDeploymentError struct {
+	Err error
+	interpreter.LocationRange
+}
+
+func (*InvalidContractDeploymentError) Error() string {
+	return "cannot deploy invalid contract"
```

Comment:
```suggestion
	return "cannot deploy invalid contract: " + e.Err
```
Do we want to include the child error message?

Reply:
```suggestion
	return fmt.Sprintf("cannot deploy invalid contract: %s", e.Err.Error())
```